travis-muesli-origDb.sql

```diff
@@ -1,1110 +1,1112 @@
---

```

Comment:
I'll have to find out which lines actually changed here. This diff is awefully large, but I suppose only some specific lines have changed.

Reply:
As suggested by you previous review we renamed the `api.sql` to overwrite the `travis-muesli-origDb.sql`.
It only contains the addition of one user as change.

Reply:
Ah, I looked at it and it seems there actually are a lot of changes. Probably due to different PostgreSQL versions. Let's keep it like it is :+1: